ansible/roles/common_installs/tasks/main.yml

```diff
@@ -58,6 +58,7 @@
     - libfreetype6-dev
     - gradle
     - ranger  # cmdline file browser
+    - libffi-dev  # Required for installing github3py
```

Comment:
Yeah, size is not remotely the issue. Putting stuff on servers that is not needed is a bad practice for other reasons: it makes maintenance more difficult, can result in implicit/undocumented dependencies over time, and in some cases can introduce security vulnerabilities.